### ISSUE TITLE
Bug in Settings Service with user time zones settings

### ISSUE THREAD

**Issue by TanyaEf** [Jaspersoft/jrs-rest-java-client#91]
The method getEntity() returns null,
```java
 final Map settings = session
                .settingsService()
                    .settings()
                        .group(USER_TIME_ZONES)
                             .getEntity();
```
as far as the server's response is:
```
HTTP/1.1 200 OK
Server: Apache-Coyote/1.1
Cache-Control: private
Expires: Thu, 01 Jan 1970 02:00:00 EET
P3P: CP="ALL"
Set-Cookie: userLocale=en_US;Expires=Tue, 14-Jul-2015 08:08:03 GMT;HttpOnly
Cache-Control: no-transform, max-age=86400
Vary: Accept-Language
Content-Type: application/json
Transfer-Encoding: chunked
Date: Mon, 13 Jul 2015 08:08:02 GMT
```
```javascript
[
  {
    "code": "Europe/Helsinki",
    "description": "Eastern European Time"
  },
  {
    "code": "America/Los_Angeles",
    "description": "Pacific Standard Time"
  },
  {
    "code": "America/Denver",
    "description": "Mountain Standard Time"
  },
  {
    "code": "America/Chicago",
    "description": "Central Standard Time"
  },
  {
    "code": "America/New_York",
    "description": "Eastern Standard Time"
  },
  {
    "code": "Europe/London",
    "description": "Greenwich Mean Time"
  },
  {
    "code": "Europe/Berlin",
    "description": "Central European Time"
  },
  {
    "code": "Europe/Bucharest",
    "description": "Eastern European Time"
  }
]
```

**Comment by Krasnyanskiy** [comment]
It is a duplicate of issue #90.

**Comment by Krasnyanskiy** [comment]
Fixed in 6.0.4.
